src/ws-week-picker/ws-week-picker-calendar.js

```diff
@@ -0,0 +1,236 @@
+import {React, Component} from '../imports';
+
+/**
+ * Quick array of all month abriviations
+ */
+const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
+/**
+ * Quick array of all month shown. There are two from the last year and two from the next year.
+ */
+const allMonths = [months[10], months[11]].concat(months).concat([months[0], months[1]]);
+
+/**
+ * Renders a week picker calendar component.
+ */
```

Comment:
I would appreciate if you could add a short description about the props maybe? Like I did for WSHeader ß